sdk/webpubsub/Microsoft.Azure.Functions.Worker.Extensions.WebPubSub/src/Models/Output/CloseGroupConnectionsAction.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Operation to close connections in a group.
+    /// </summary>
+    public sealed class CloseGroupConnectionsAction : WebPubSubAction
+    {
+        /// <summary>
+        /// Target group name.
+        /// </summary>
+        public string Group { get; set; }
+
+        /// <summary>
+        /// ConnectionIds to exclude.
+        /// </summary>
+        public IList<string> Excluded { get; set; } = new List<string>();
```

Comment:
This goes against the SDK convention of having collection-type members for models as get-only.  Please discuss with @KrzysztofCwalina and @tg-msft.

Reply:
The isolated process extension needs these types to do Json deserialize and requires a parameterless constructor that not what we want to have in the Common package. And these types are specifically used in this isolated process project.
The trigger types are essential to declare and rest types are simply easier for customer to use in the isolated functions. Otherwise, they have to define these themselves.